x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_creation/custom_query_rule.cy.ts

```diff
@@ -352,13 +350,9 @@ describe('Custom query rules', { tags: [tag.ESS, tag.BROKEN_IN_SERVERLESS] }, ()
       const expectedEditedtags = rule.tags?.join('');
       const expectedEditedIndexPatterns = rule.index;
 
-      before(() => {
-        deleteAlertsAndRules();
```

Comment:
Why are we deleting this method from the hook?

Reply:
`deleteAlertsAndRules()` has been added on line `125` inside `beforeEach` hook so we don't need to invoke the same function here twice.

Reply:
Gotcha!! Great refactor there :)